specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -218,23 +229,30 @@ function ConditionalFormatter({
         />
       ) : null}
       <span className="-ml-2 flex-1" />
-      {trimmedFieldsLength === 1 || isExpanded || isReadOnly ? null : (
-        <div className="inline-flex">
-          <Button.Icon
-            icon="trash"
-            title={resourcesText.deleteDefinition()}
-            onClick={handleDelete}
-          />
-        </div>
-      )}
-      <div className="flex">
-        {hasCondition && !isExpanded ? (
-          <Button.Icon
-            icon="chevronDown"
-            title={resourcesText.expandConditionalField()}
-            onClick={handleToggle}
-          />
+      <div className="flex flex-col p-2">
+        {!isExpanded && index === 0 ? (
+          <span className="font-bold">{commonText.expand()}</span>
         ) : null}
+        <div className="flex justify-end">
```

Comment:
(optional)
I don't think `justify-end` is needed here. 
The elements being "right justified" should be handled by the above span with `-ml-2`, correct? 
Unless there is some other reason `justify-end` is used here?

Reply:
the justify end is needed here to keep the arrow at the same place when there is no trash icon
see with and without

https://github.com/specify/specify7/assets/108160931/22ec8bc0-f6a0-41e9-8d3f-628b4a990013


https://github.com/specify/specify7/assets/108160931/780be95b-da6e-4dda-82d5-ba6fbb9a5636